cmd/list/listtopic.go

```diff
@@ -91,6 +105,14 @@ func (l *listTopic) listLastWrittenTopics() error {
 	return nil
 }
 
+func (l *listTopic) filterEmptyTopicsIfNeeded() ([]string, error) {
+	if l.isEmpty {
+		return l.ListEmptyLastWrittenTopics(l.lastWrite, l.dataDir)
```

Comment:
let's not club lastwrite and empty, as i can choose to just list empty topics as of now, and if we pass lastwritten flag then filter. Also this avoid need for ssh (lastwritten)

Reply:
We needed to implement default as 2 weeks when the empty flag is set, did we not